src/AgStateMachine.cpp

```diff
@@ -302,20 +302,24 @@ void StateMachine::co2Calibration(void) {
 
 void StateMachine::ledBarTest(void) {
   if (config.isLedBarTestRequested()) {
-    if (config.getCountry() == "TH") {
-      uint32_t tstart = millis();
-      logInfo("Start run LED test for 2 min");
-      while (1) {
-        ledBarRunTest();
-        uint32_t ms = (uint32_t)(millis() - tstart);
-        if (ms >= (60 * 1000 * 2)) {
-          logInfo("LED test after 2 min finish");
-          break;
-        }
-      }
-    } else {
+    ledBarPowerUpTest();
+  }
+}
+
+void StateMachine::ledBarPowerUpTest(void) {
+  if (config.getCountry() == "TH") {
```

Comment:
I think we shouldn't have a different behavior based on the country setting

Reply:
There are 3 inputs parameter: `StateMachine`, `Configuration`, and `*AirGradient`. 

As you say we can pass `Configuration`? Or should remove alls?